src/types/inventory/deliveryOrder/deliveryOrder.ts

```diff
@@ -3,8 +3,9 @@ import { LocationDetail } from "@/types/location/location";
 import { Organization } from "@/types/organization/organization";
 
 export enum DeliveryOrderStatus {
-  draft = "draft",
   pending = "pending",
+  in_progress = "in_progress",
```

Comment:
I did remove `in_progress` intentionally, not sure what could be implied with `in_progress` in this case.